backend/app/data/test/wikipedia_arkg.ttl

```diff
@@ -1,9 +1,16 @@
-<http://imlapps.github.io/anti-recommender/anti-recommendation/Nikola_Tesla> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/Recommendation> .
-<http://imlapps.github.io/anti-recommender/anti-recommendation/Nikola_Tesla> <http://schema.org/itemReviewed> <http://www.wikidata.org/entity/Q44666> .
-<http://www.wikidata.org/entity/Q44666> <http://schema.org/url> "http://www.wikidata.org/entity/Q44666" .
-<http://www.wikidata.org/entity/Q44666> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/WebPage> .
-<http://www.wikidata.org/entity/Q44666> <http://schema.org/title> "Laplace's_demon" .
-<http://imlapps.github.io/anti-recommender/anti-recommendation/Nikola_Tesla> <http://schema.org/itemReviewed> <http://www.wikidata.org/entity/Q762> .
-<http://www.wikidata.org/entity/Q762> <http://schema.org/url> "http://www.wikidata.org/entity/Q762" .
-<http://www.wikidata.org/entity/Q762> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/WebPage> .
-<http://www.wikidata.org/entity/Q762> <http://schema.org/title> "Leonardo_da_Vinci" .
\ No newline at end of file
+@base <http://imlapps.github.io/anti-recommender/anti-recommendation/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix schema: <http://schema.org/> .
+@prefix wd: <http://www.wikidata.org/entity/> .
+
+<Nikola_Tesla> rdf:type schema:Recommendation .
```

Comment:
(UUID identifier, rdf:type, schema:Recommendation)

https://datatracker.ietf.org/doc/html/draft-kindel-uuid-uri-00

Reply:
I have changed the structure of the Wikipedia ARKG.